axon-server-connector/src/main/java/org/axonframework/axonserver/connector/command/CommandLoadFactorProvider.java

```diff
@@ -0,0 +1,24 @@
+package org.axonframework.axonserver.connector.command;
+
+/**
+ * Provides the load factor value of the client for the specific command.
+ * This information is used from AxonServer in order to balance the load among client instances.
+ *
+ * @author Sara Pellegrini
+ * @since 4.3
+ */
+public interface CommandLoadFactorProvider {
+
+    /**
+     * The default value for command load factor.
```

Comment:
Nit: Shouldn't this tiny bit of javadoc also state what the default is?

Reply:
yes, right. Take a look if it's clearer now

Reply:
yes, right. Take a look if it is clearer now

Reply:
Ah, what I meant is that I would assume it states the number `100` ^_^;

Reply:
Done